cookbooks/crystal_preferred_orientation_olivine_fraters_billen_2021/doc/crystal_preferred_orientation_olivine_fraters_billen_2021.md

```diff
@@ -0,0 +1,119 @@
+# Olivine Fabric Developments Under Simple Shear
+
+*This section was contributed by Xiaochuan Tian, with helps from Yijun Wang and Menno Fraters. It is
+based on a section in {cite:t}`fraters_billen_2021_cpo`
+by Menno Fraters and Magali Billen published in 2021.*
+
+This cookbook explains how to set up a numerical experiment for fabric
+developments of a single olivine particle under simple shear macroscopic strain.
+It uses ASPECT's
+crystal preferred orientation (CPO) implementation, which is described in detail in
+{cite:t}`fraters_billen_2021_cpo`.
+The fabric calculation is based on DRex {cite:t}`kaminski2004`.
+This cookbook describe how to use ASPECT CPO implementation to calculate the major
+olivine fabrics observed in the lab experiments. It focuses on reproducing Fig. 3
+of {cite}`fraters_billen_2021_cpo`.
+
+## Motivation
+The Earth's plates can move relative to each other because the underlying mantle
+can be deformed to accommodate such motions.
+This motion reflects dynamics inside the planet but cannot be observed directly, as the depth of interest is beyond human drilling capability. However, indirect observations from seismic anisotropy provide information about the directionality of seismic wave speeds propagating inside the Earth. If such directionality is controlled by the mantle’s motion or deformation, we can use seismic anisotropy observations to infer mantle deformation.
+
+It’s important to note that this cookbook does not directly compare model outputs with seismic anisotropy observations. Instead, it provides the full elastic tensor, including its symmetry axes and fast directions, at the locations of particles in the model. To make direct comparisons with seismic observations, you would still need to process these results through a separate code that simulates seismic wave propagation through the modeled medium.
+
+Olivine is a major mineral of the Earth's upper mantle where continuous deformation
+takes place. People care about olivine fabric developments under simple shear as it may
+provide a bridge that links seismic anisotropy observations to the
+flows in the upper mantle that accommodate plate motions. Indeed, high temperature and
+pressure lab experiments using [Griggs apparatus](https://en.wikipedia.org/wiki/Griggs_apparatus) to investigate olivine fabrics developments
+under simple shear do find systematic types of fabrics based on the stress and water content conditions
+(see {cite:t}`karato2008` for details). Note that the effects of secondary phases, such as enstatite  {cite:t}`kaminski2004` can also be included in the method, we here only focus on olivine for simplicity and the input file also include example of enstatite calculation setup.
```

Comment:
This sentence seems to contradict itself. First you say we only focus on olivine, then you say the input file also includes enstatite. Which of the two is it?